content/blueprints/gcp.md

```diff
@@ -0,0 +1,1277 @@
+  * `health_check`
+    URL of a health check assigned to this backend service.
+
+    *type:* string
+    *default:*
+  * `use_external_resource`
+    Indicate whether the resource exists and use existing (true) or if Cloudify should create new resource (false).
+
+    *type:* boolean
+    *default:* False
+  * `name`
+    Optional health check name. By default it will be backend service id.
+
+    *type:* string
+    *default:*
+
+
+
+
+
+## cloudify.gcp.nodes.DNSAAAARecord
+**Derived From:** cloudify.gcp.nodes.DNSRecord
+
+`AAAA` type DNSRecord
+
+
+
+**Properties:**
+
+
+  * `type`
+
+
+    *default:* AAAA
+
+
+
+
+
+## cloudify.gcp.nodes.DNSMXRecord
+**Derived From:** cloudify.gcp.nodes.DNSRecord
+
+`MX` type DNSRecord
+
+
+
+**Properties:**
+
+
+  * `type`
+
+
+    *default:* MX
+
+
+
+
+
+## cloudify.gcp.nodes.DNSNSRecord
+**Derived From:** cloudify.gcp.nodes.DNSRecord
+
+`NS` type DNSRecord
+
+
+
+**Properties:**
+
+
+  * `type`
+
+
+    *default:* NS
+
+
+
+
+
+## cloudify.gcp.nodes.DNSRecord
+**Derived From:** [cloudify.nodes.Root]({{< relref "blueprints/built-in-types.md" >}})
+
+Corresponds to a particular subdomain (or `@` for the root) and record-type in the containing DNSZone.
+
+e.g. the `A` record for `special_service.getcloudify.org`
+
+A number of convenience types are provided which update the default type (see DNSAAAARecord, DNSMXRecord, DNSTXTRecord, DNSNSRecord)
+
+
+
+**Properties:**
+
+
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the GCP API.
+
+    *default:* {}
+  * `type`
+    The type of this DNS record. Only one record of each type is allowed with the same name within a zone.
+
+    *type:* string
+    *default:* A
+  * `name`
+    The subdomain. This will be prepended to the DNSZone's dns_name to produce the full domain name for this record. Defaults to the instance ID.
+
+    *type:* string
+    *default:*
+  * `resources`
+    List of resources which will form this record. (can be augmented using relationships
+      cloudify.gcp.relationships.dns_record_connected_to_instance
+      and
+      cloudify.gcp.relationships.dns_record_connected_to_ip
+      )
+
+    *default:* []
+  * `ttl`
+    DNS entry Time To Live
+
+    *type:* integer
+    *default:* 86400
+
+
+
+### Example
+
+{{< gsHighlight  yaml  >}}
+
+www:
+  type: cloudify.gcp.nodes.DNSRecord
+  properties:
+    resources: [10.11.12.13, 8.9.10.11]
+  relationships:
+    - type: cloudify.gcp.relationships.dns_record_contained_in_zone
+      target: my_zone
+
+mx:
+  type: cloudify.gcp.nodes.DNSMXRecord
+  properties:
+    name: mail
+  relationships:
+    - type: cloudify.gcp.relationships.dns_record_contained_in_zone
+      target: my_zone
+    - type: cloudify.gcp.relationships.dns_record_connected_to_instance
+      target: my_instance
+
+{{< /gsHighlight >}}
+
+The DNSRecord type can be connected to an instance or directly to an IP. In this case the (associated) public IP will be added to the list of resources.
```

Comment:
Again, for consistency, *IP* should be *ip*. My preference is actually for acronyms in comments to be capitalized (best practice), but more important is that there is consistency.